src/ralph/admin/widgets.py

```diff
@@ -189,92 +189,32 @@ def render(self, name, value, attrs=None):
                 ),
                 current_app=self.admin_site.name,
             ) + self.get_url_parameters()
-            searched_fields = []
-            searched_fields_tooltip = defaultdict(list)
-            polymorphic_descendants = getattr(
-                self.rel_to, '_polymorphic_descendants', []
-            )
 
-            if polymorphic_descendants:
```

Comment:
this logic couldn't be removed IMO - new widget should support polymorphic models too.